operator/watchers/k8s_service_sync.go

```diff
@@ -136,7 +125,7 @@ func StartSynchronizingServices(ctx context.Context, wg *sync.WaitGroup, cfg Ser
 		<-kvstoreReady
 
 		log.Info("Starting to synchronize Kubernetes services to kvstore")
-		k8sServiceHandler(ctx, cfg.LocalClusterName(), cfg.SharedOnly, cfg.LocalClusterID())
+		k8sServiceHandler(ctx, cfg.ClusterName, cfg.SharedOnly, cfg.ClusterID)
```

Comment:
Perhaps we could streamline args in these calls and pass in ClusterIDName instead? if considered in scope with this PR or later refactor...